doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+the same projection cell to be combined into larger mosaics as part of the same tangent plane without performing any
+additional resampling.
+
+Code for Defining SkyCell ID
+'''''''''''''''''''''''''''''
+The code for interfacing with the cell definitions table can be imported in Python using:
+
+.. code:: python
+
+   from drizzlepac.haputils import cell_utils
+
+Determining what sky cells overlap any given set of exposures on the sky can be done using the function:
+
+.. code:: python
+
+  sky_cells_dict = cell_utils.get_sky_cells(visit_input, input_path=None)
+
+where **visit_input** is the Python list of filenames of exposures.
+Exposures in an input list are assumed to be in the current working directory when running the code,
+unless **input_path** has been provided which points to the location of the exposures to be processed.
+The return value **sky_cells_dict** is a dictionary where the keys are the names (labels) of each overlapping sky cell
+and the value is the actual SkyCell object which contains the footprint and WCS (among other details) of the sky cell.
+
+
+Defining SkyCell Layers
+------------------------
+Defining the SkyCell for a region on the sky allows for the identification of all exposures that overlap that WCS.
+However, creating a single mosaic from data taken with different detectors and filters would not result in a
+meaningful result.  Therefore, the exposures that overlap each SkyCell get grouped based on the detector and filter used
+to take the exposure to define a 'layer' of the SkyCell.  Each layer can then be generated as the primary basic image
+product for each SkyCell.  Exposures taken with spectroscopic elements, like grisms and prisms, and exposures taken of
+moving targets can not be used to create layers due to the inability to align them with the rest of the observations.
+Therefore, only images taken with standard filters (like the WFC3/UVIS F275W filter) will be used to define SkyCell
+mosaics (layers).
+
+The default plate scale for all MVM image products for each SkyCell has been defined as 0.04"/pixel to match the higher
+resolution imaging performed by the WFC3/UVIS detector.  However, WFC3/IR data suffers from serious resampling artifacts
+when drizzling IR data to that plate scale. So in addition to creating IR mosaics at the 0.04"/pixel 'fine' plate scale,
+IR mosaics are also generated at a 'coarse' plate scale of 0.12"/pixel to minimize the resampling artifacts while also being easily
+scaled to the 'fine' plate scale mosaics.
+
+SkyCell Example
+'''''''''''''''
+For example, observations have been taken of NGC5474 with both the ACS and WFC3 cameras.  The ACS observations were taken
+with the ACS/WFC detector using the F814W and F606W filters, while the WFC3 observations were taken using the IR detector
+using the F110W and F160W filters.  All these observations fall within the SkyCell at position X06y11 in the
```

Comment:
Change "X06y11" to have consistent case.

Reply:
Changed in 241a672